app/move/controllers/update/assessment.js

```diff
@@ -0,0 +1,71 @@
+const { get, keys, pick } = require('lodash')
+
+const personService = require('../../../../common/services/person')
+const Assessment = require('../create/assessment')
+
+const UpdateBase = require('./base')
+
+const compare = (a, b) => {
+  const key = 'assessment_question_id'
+  if (a[key] < b[key]) {
+    return -1
+  }
+  if (a[key] > b[key]) {
+    return 1
+  }
+}
+const getAnswerKeys = answers => {
+  return answers
+    .map(x => pick(x, ['key', 'assessment_question_id', 'comments']))
+    .sort(compare)
+}
+
+class UpdateAssessmentController extends UpdateBase {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setPreviousAssessment)
+  }
+
+  async saveValues(req, res, next) {
+    try {
+      const personId = req.getPersonId()
+      const person = req.getPerson()
+      const assessments = person.assessment_answers || []
+      const fieldKeys = keys(get(req, 'form.options.fields'))
```

Comment:
Could be `const fieldsKeys = Object.keys(req.form.options.fields)` and do away with needing the lodash methods.

And to be honest, if `req.form.options.fields` isn't there we kind of don't mind this all blowing up as there's likely a bigger issue.

Reply:
Done